### DISCUSSION TITLE
YAML file format

### DISCUSSION THREAD

**Discussion by s-canchi** [nih-cfde/training-and-engagement#363] [Q&A] [answered]
What is YAML and what programs can you use to create a .yml file ?

**Comment by s-canchi** [comment, marked as answer]
- YAML Ain't Markup Language (YAML) is a human friendly data serialization standard for all programming languages.
- You can make `.yml` file in any text editor program! 
-  YAML format is nice because it’s both computer and human-readable.

**Reply by ctb** [reply]
https://en.wikipedia.org/wiki/YAML and https://yaml.org/ are good references.
